contracts/vaults/DnGmxJuniorVault.sol

```diff
@@ -234,6 +234,14 @@ contract DnGmxJuniorVault is IDnGmxJuniorVault, ERC4626Upgradeable, OwnableUpgra
         );
     }
 
+    /// @notice set thresholds
+    /// @param rebalanceProfitUsdcAmountThreshold (BPS) slippage threshold on btc swaps
+    function setThresholdsV1(uint128 rebalanceProfitUsdcAmountThreshold) external onlyOwner {
```

Comment:
maybe we should use V2 or V1.1 in naming? as per the current npm package semver, v1 is what is currently deployed.

Reply:
also `setThresholdsV1` is not called anywhere in fixtures, so 0 is used everywhere in all test cases (i.e same behavior without this logic change), so we should test using non-zero values covering all paths